pkg/cache/cache_test.go

```diff
@@ -32,10 +32,10 @@ func TestExpireRegionCache(t *testing.T) {
 	// Test Pop
 	cache.PutWithTTL(9, "9", 5*time.Second)
 	cache.PutWithTTL(10, "10", 5*time.Second)
-	re.Equal(2, cache.Len())
+	re.Len(cache, 2)
```

Comment:
`.Len()` is a method of `cache`, it's not the `re.Len` can only be used for Array, Chan, Map, Slice, or String.